File: day12/Test8.java
package com.day12;

import java.util.Vector;
//Vector안에는 뭐든 다들어간다.배열 안에도 뭐든 다들어간다.

class Test {
	
	String name;
	int age;
}

public class Test8 {

	public static void main(String[] args) {
		
		Vector<Test> v = new Vector<>(); // Vector에 Test클래스를 넣는다.
		
		Test ob;
		ob = new Test();	// 2명의 데이터를 넣어서 2번씀
		ob.name = "배수지";
		ob.age = 27;
		v.add(ob);
		
		ob = new Test();// 2명의 데이터를 넣어서 2번씀
		ob.name = "유인나";
		ob.age = 41;
		v.add(ob);
		
		for(Test t : v) {
			System.out.println(t.name + ":" + t.age);
		}
		
	}

}

File: React_basic/test2/src/componets/Test8.js
import React from 'react';

const Test8 = () => {

    const title = '연예인'
    const arr = ['배수지','유인나','정인선','강아랑','이영애']
    const data = [
        {id:1,name:'배수지',age:27},
        {id:2,name:'유인나',age:25},
        {id:3,name:'정인선',age:21},
        {id:4,name:'강아랑',age:24},
        {id:5,name:'이영애',age:20},
    ]

    return (
        <div>
            <h2>{title}</h2>
            <p>{arr[0]}</p>
            <p>{arr[3]}</p>

            {
                arr.map((item,index)=> {
                    return (<p key={index}>{index}/{item}</p>)
                })
            }
            <hr/>
            {
                arr.map((item,index)=><p key={index}>{index}/{item}</p>)
            }
            <hr/>
            {
                data.map((item,index)=>{
                    return(<p key={index}>{item.id}/{item.name}/{item.age}</p>)
                })
            }
            <hr/>
            {
                data.map((item,index)=><p key={index}>{item.id}/{item.name}/{item.age}</p>)
            }

        </div>
    );
};

export default Test8;